src/App.js

```diff
@@ -1,15 +1,20 @@
-import React from 'react';
-import TodoList from './TodoList';
-import AddTodoForm from './AddTodoForm';
+import React from "react";
+import AddTodoForm from "./AddTodoForm";
+import TodoList from "./TodoList";
 
 function App() {
-  const [newTodo, setNewTodo] = React.useState();
+
+  function addTodo(newTodo) {
+    setTodoList([...todoList, newTodo]);
+  }
+
+  const [todoList, setTodoList] = React.useState([]);
```

Comment:
By convention, it is best to declare the useState hook right at the beginning of the function.

Reply:
<img width="610" alt="app" src="https://github.com/mvstoyan/react-todo/assets/95896159/a7647a77-67e6-44f1-a5e4-a0170d2ce400">